src/onramp.rs

```diff
@@ -89,6 +89,7 @@ pub(crate) fn lookup(
         "otel" => otel::OpenTelemetry::from_config(id, config),
         "nats" => nats::Nats::from_config(id, config),
         "gsub" => gsub::GoogleCloudPubSub::from_config(id, config),
+        "unix-socket" => unix_socket::UnixSocket::from_config(id, config),
```

Comment:
we use it for pre and post processors, so perhaps it's alright? socket could be is a bit confusing since a lot of the connectors use sockets at one point or another